src/main/java/foundation/privacybydesign/email/EmailRestApi.java

```diff
@@ -134,7 +134,7 @@ public Response sendEmailToken(@FormParam("email") String emailAddress,
     public Response verifyEmailToken(@FormParam("token") String token) throws KeyManagementException {
         EmailConfiguration conf = EmailConfiguration.getInstance();
 
-        String emailAddress = signer.verifyToken(token);
+        String emailAddress = signer.verifyToken(token).toLowerCase();
```

Comment:
Maybe it is neat then to also lowercase the email address in the confirmation email to prevent inconsistencies there.